### PR TITLE
Hamburgers changes made logo added

### PR DESCRIPTION
**Pull request by meharwalia**

### CODE REVIEW

**Review by Siddharth-Agg-arwal — CHANGES_REQUESTED** [review]
fix the mlsc logo position on the sponsors page (place it on the left)

**Comment by MankiratSingh1315** [comment]
Disable scroll when hamburger is opened

File: app/page.module.css [hunk 1]
```diff
@@ -300,17 +337,21 @@
...
 }
-
-@media  screen and (min-width: 1383px){
+@media screen and (max-height: 1600px){
     .throne{
-        top: 15vh;
+        top: 13.9vh;
     }
-} */
+
```
Comment (MankiratSingh1315) [inline app/page.module.css]:
```suggestion
}
```

**Review by MankiratSingh1315 — DISMISSED** [review]

**Review by Siddharth-Agg-arwal — APPROVED** [review]

**Comment by Siddharth-Agg-arwal** [comment]
changes reviewed

**Pull request merged**